Assets/Prefabs/Panels/MultiplayerPanel.prefab

```diff
@@ -1399,10 +1400,8 @@ MonoBehaviour:
   m_OnCullStateChanged:
     m_PersistentCalls:
       m_Calls: []
-  m_text: 'You are in!
-
-    Share the room number with your sketch collaborators
-    to join '
+  m_text: You have successfully joined the room! Share the room number with your
```

Comment:
Be aware we need to handle localization for all text. Even if we only have an English translation for now - the components and code should support localization.